### DISCUSSION TITLE
Nesting VMs with Vagrant inside of VSphere ESXi?

### DISCUSSION THREAD

**Discussion by peroxidee** [nasa/nos3#245] [Q&A]
I am having trouble with nesting the machine that vagrant spawns. It is running on an Ubuntu VM inside of VSphere/VCenter, and keeps returning the error ``AMD-V is not Avaliable (VERR_SVM_NO_SVM)``.  I don't have the option to enable it through BIOS, is there any other options anyone would suggest?

**Comment by jlucas9** [comment]
Hi @peroxidee,
There is the option to install directly on your VSphere/VCenter VM if you have permissions to do that and avoid the nesting virtualization. We provision the NOS3 base boxes using the https://github.com/nasa-itc/deployment repository. There is a [deploy-local-env.sh](https://github.com/nasa-itc/deployment/blob/main/deploy-local-env.sh) script that can be used if you're running Ubuntu or modified to support Rocky / RHEL. 
Let me know if that gives you any other issues. I'm happy to help!
